Jenkinsfile

```
@@ -0,0 +1,10 @@
+@Library('global-jenkins-library@2.1.1') _
+
+node('docker') {
+    buildInfo = getBuildInfo()
+    buildSimpleDocker_v3(
+        buildInfo: buildInfo,
+        imageprivacy: 'private',
+        dockerImageRepositoryName : 'netstats'
+    )
+}
```

```suggestion
    buildInfo = getBuildInfo()
    buildSimpleDocker_v3(
        buildInfo: buildInfo,
        imageprivacy: 'private',
        dockerImageRepositoryName : 'netstats'
    )
```
No need to specify Jenkins node here as it is specified in the library